torchci/pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -102,7 +102,8 @@ function HudRow({
           <a
             href={`https://github.com/${params.repoOwner}/${params.repoName}/pull/${rowData.prNum}`}
           >
-            #{rowData.prNum}
+            {rowData.isForcedMerge && (<mark>#{rowData.prNum}</mark>)}
+            {!rowData.isForcedMerge && (<div>#{rowData.prNum}</div>)}
```

Comment:
nit, can you do something like:

```suggestion
            {rowData.isForcedMerge ? (<mark>#{rowData.prNum}</mark>) : (<div>#{rowData.prNum}</div>)}
```